### PR TITLE
Customize tile layout

### PR DESCRIPTION
**Pull request by dlackty**
For some use cases, library user might want to swap the layout for image grid item.

### CODE REVIEW

**Comment by dlackty** [comment]
@Taresin Just rebased the PR on latest master. Could you please help review the changes? Also I would like to sign the CLA but the link on README leads to 404.

**Comment by dlackty** [comment]
@markrietveld Hello, any chance to get this reviewed and merged in? I thought it's quite trivial change and would make this awesome library even more flexible.

**Comment by markrietveld** [comment]
@dlackty sorry for the silence leading up to this. I missed this pull request.

What is the use case for this change? The inflated layout needs to be an ImageView, or at least extend ImageView. That doesn't provide a lot of flexibility. Depending on what you're trying to achieve, there might be a better way to do it.

**Comment by dlackty** [comment]
Thanks for the response! What we want to achieve is to use BottomSheet with Facebook's [Fresco](http://frescolib.org/) library. i.e. Replace the default tile with [`SimpleDraweeView`](http://frescolib.org/javadoc/reference/com/facebook/drawee/view/SimpleDraweeView.html), which is an `ImageView` subclass.

File: bottomsheet-commons/src/main/java/com/flipboard/bottomsheet/commons/ImagePickerSheetView.java [hunk 1]
```diff
@@ -501,6 +509,18 @@ public Builder setPickerDrawable(@DrawableRes int resId) {
         }
 
         /**
+         * Sets a layout for the image tile. Default is to use plain image view included in the
+         * library.
```
Comment (markrietveld) [inline bottomsheet-commons/src/main/java/com/flipboard/bottomsheet/commons/ImagePickerSheetView.java]: You should add that the root of the layout must be an `ImageView`, or the sheet will crash at runtime.

Comment (dlackty) [reply bottomsheet-commons/src/main/java/com/flipboard/bottomsheet/commons/ImagePickerSheetView.java]: @markrietveld Resolved and also added a type check.

**Comment by markrietveld** [comment]
@dlackty thanks for contributing!

**Comment by dlackty** [comment]
Thanks @markrietveld!

**Pull request merged**